syntax = "proto3";

option go_package = "api/backtest";

package backtest;

message RunBacktestReq {
    string code = 1;
}

message RunBacktestRes {
    string code = 1;
    bool result = 2;
}

service Backtest {
    rpc RunBacktest (RunBacktestReq) returns (RunBacktestRes);
    rpc RunBacktestStream (stream RunBacktestReq) returns (stream RunBacktestRes);
}